src/d3/scatterplot.d3.ts

```diff
@@ -21,20 +22,12 @@ export class Scatterplot{
     private dataTextContainer: HTMLElement;
     private metrics: string[];
     private cardSelection: any;
-    private DAYDREAM_NAV_SPEED;
+    DAYDREAM_NAV_SPEED;
```

Comment:
Can make this private, then for `createSpeedCtrls`, can pass in a setter function:

createSpeedCtrls(sign, incrementSpeedFn);

Usage inside `createSpeedCtrls`:

Instead of:
```
scatter.DAYDREAM_NAV_SPEED = scatter.DAYDREAM_NAV_SPEED + .2;
```

You would do:
```
incrementSpeedFn(.2);
```